spec/spec_helper.rb

```diff
@@ -23,6 +23,7 @@
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 # include the manageiq-gems-pending matchers
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/i18n/*.rb")].each { |f| require f }
```

Comment:
Upon reviewing this more and seeing the origin of `/shared` (https://github.com/ManageIQ/manageiq/pull/11414) I think your original line was correct. Let's change this back to just `/shared/**/*.rb` and place it right below line 23 (the support one). Then all shared examples here will be available to the core specs (and we'll start consolidating them there, more.